bindings/java/Swig/post-build.cmd

```diff
@@ -30,7 +30,7 @@ if "%is_gpu%" == "true" (
   )
 )
 
-for %%x in (libiomp5md.dll mklml.dll mkldnn.dll Cntk.Math-%version%.dll Cntk.PerformanceProfiler-%version%.dll Cntk.Core-%version%.dll Cntk.Core.JavaBinding-%version%.dll) do (
+for %%x in (opencv_world310.dll libiomp5md.dll mklml.dll mkldnn.dll Cntk.Math-%version%.dll Cntk.PerformanceProfiler-%version%.dll Cntk.Core-%version%.dll Cntk.Core.JavaBinding-%version%.dll) do (
```

Comment:
zip.dll and zlib.dll are also required along with opencv. Please add them as well.
Note that these dlls are optional. So, make sure copy operation doesn't fail if these dlls are not present.

Reply:
Done